code/modules/mob/living/carbon/examine.dm

```diff
@@ -96,9 +96,9 @@
 	if(HAS_TRAIT(src, TRAIT_DUMB))
 		msg += "[t_He] seem[p_s()] to be clumsy and unable to think.\n"
 
-	if(fire_stacks > 0)
+	if(get_fire_stacks() > 0)
 		msg += "[t_He] [t_is] covered in something flammable.\n"
-	if(fire_stacks < 0)
+	if(get_fire_stacks() < 0)
```

Comment:
Also, IMO you should handle these via the status effect's `examine_text` instead. Though I refactored it in #66340 so you may want to wait for that, since it'll make it easier.